docs/sdk/python/_implement_sign_out_route.mdx

```diff
@@ -0,0 +1,24 @@
+import postSignOutRedirectUriFigure from '../assets/go-post-sign-out-redirect-uri.png';
+import ConfigurePostSignOutRedirectUri from '../fragments/_configure-post-sign-out-redirect-uri.mdx';
+
+After the user makes a signing-out request, Logto will clear all user authentication information in the session.
+
+To clean up the Python session and Logto session, a sign-out route can be implemented as follows:
+
+<ConfigurePostSignOutRedirectUri
+  figureSrc={postSignOutRedirectUriFigure}
+  postSignOutRedirectUri="http://localhost:8080"
+/>
+
+```python
+@app.route("/sign-out")
+async def sign_out():
+    return redirect(
+        # Redirect the user to the home page after a successful sign-out
+        await client.signOut(postLogoutRedirectUri="http://localhost:8080")
+    )
+```
+
+`postLogoutRedirectUri` is optional, and if not provided, the user will be redirected to a Logto default page after a successful sign-out (without redirecting back to your application).
+
+> The name `postLogoutRedirectUri` is from the [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html) specification. Although Logto uses "sign-out" instead of "logout", the concept is the same.
```

Comment:
Either remove this section or create a general component to align with other tutorials @xiaoyijun any ideas?

Reply:
Good call, wrapped this part as component <ImplementSignOutRouteNotice /> for later use